modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -106,6 +107,21 @@ public String getLabel(Locale locale) {
 		return _language.get(locale, "data-set");
 	}
 
+	public JSONArray getSortsJSONArray(
+			ObjectEntry fdsView, ObjectDefinition objectDefinition)
+		throws Exception {
+
+		List<ObjectEntry> fdsSortingObjectEntries = new ArrayList<>(
+			_getRelatedObjectEntries(
+				objectDefinition, fdsView, "fdsViewFDSSortRelationship"));
+
+		if (ListUtil.isEmpty(fdsSortingObjectEntries)) {
+			return _jsonFactory.createJSONArray();
+		}
+
+		return _getSortsJSONArray(fdsSortingObjectEntries);
+	}
```

Comment:
Thanks @markocikos, I've merged it into a private method. 

One doubt that come up was: the parent public method uses the `thrown exception` clause, and in the other hand the private one uses a try/catch block inside. As far as I understand (probably because my limited knowledge in java) those two things doesn't make sense together, and I got rid of the `thrown exception`. The reason was the try/catch block were not throwing and exception but returning and empty json.